### PR TITLE
Feature/update communication functions

### LINKED ISSUE #147: Add list of clip id:s to ModelSerializers for Camera and Folder.

### LINKED ISSUE #149: Update communication functions
New
get_source_folders: gets all folders.

Update:
Add folder, should only add folder to project. Not create new one.

### PR DESCRIPTION
**Pull request by Kal-Elx**
Closes #149

### CODE REVIEW

**Comment by Kal-Elx** [comment]
Closes #147

File: backend/test/test_integration/test_file_manager.py [hunk 1]
```diff
@@ -5,6 +5,26 @@
 from backend.file_manager import *
 
 
+class GetSourceFolders(TestCase):
```
Comment (SteBry) [inline backend/test/test_integration/test_file_manager.py:8]: Add one test case for when no folders exist and one for several rootfolders

**Review by Flodklumpen — CHANGES_REQUESTED** [review]
Looks good. Just small linting-stuff.
file: model_serializers.py
line: 2
Not supposed to be more than one import on one line. 
line: 14
Should be two blank lines.

File: backend/test/test_integration/test_file_manager.py (hunk 1 above)
Comment (SteBry) [reply backend/test/test_integration/test_file_manager.py:8]: Otherwise it looks good!

**Review by Flodklumpen — APPROVED** [review]

**Pull request merged**